src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -474,7 +475,7 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
                         Sequence = jsonStepInfo.Sequence,
                         RatingTextTitle = formattedText.ToString(),
                         StepType = Step.StepTypes.SURVEY,
-                        IsRatingVisible = dynamoViewModel.Model.PreferenceSettings.IsADPAnalyticsReportingApproved,
```

Comment:
@mjkkirschner @QilongTang any ideas what this is used for ?

Reply:
This is about the five star rating after guided tour exited, when analytics opt-in, we display the rating for ppl to rate

Reply:
well now we will no longer know if ADP is opted in or not.
Should I just check for google analytics ? or as it is (if no one explicitly disabled analytics) ?

Reply:
After this change, it seems only startup params would impact which I think should apply. I would leave it as it is. If user opt-out for GA, I think the client would send nothing right?

Reply:
Sounds right